manifests/m/Microsoft/PerfView/3.0.6/Microsoft.PerfView.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.PerfView
+PackageVersion: 3.0.6
+PackageLocale: en-US
+Publisher: Microsoft
```

```suggestion
Publisher: Microsoft Corporation
```
AFAIK, all MS apps use this for the Publisher field.